aws/resource_aws_elasticache_replication_group.go

```diff
@@ -852,8 +852,11 @@ func deleteElasticacheReplicationGroup(replicationGroupID string, conn *elastica
 		}
 		return nil
 	})
+	if isResourceTimeoutError(err) {
+		_, err = conn.DeleteReplicationGroup(input)
+	}
```

Comment:
This is another case where we want to ignore the error if the resource is already deleted (as it does in the `resource.Retry()` above). 👍 

```suggestion
	}

	if isAWSErr(err, elasticache.ErrCodeReplicationGroupNotFoundFault, "") {
		return nil
	}

```